confd/templates/janitor.properties.tmpl

```diff
@@ -1,2 +1,191 @@
-# Disable Janitor Monkey
-simianarmy.janitor.enabled = false
+# see documentation at:
+# https://github.com/Netflix/SimianArmy/wiki/Configuration
+
+# By default Janitor Monkey wakes up every hour
+simianarmy.scheduler.frequency = {{getv "/simianarmy/scheduler/frequency" "1"}}
+simianarmy.scheduler.frequencyUnit = {{getv "/simianarmy/scheduler/frequencyunit" "HOURS"}}
+simianarmy.scheduler.threads = {{getv "/simianarmy/scheduler/threads" "1"}}
+# Janitor Monkey runs every day at 11am.
+simianarmy.calendar.openHour = {{getv "/simianarmy/calendar/openhour" "11"}}
+simianarmy.calendar.closeHour = {{getv "/simianarmy/calendar/closehour" "11"}}
+simianarmy.calendar.timezone = {{getv "/simianarmy/calendar/timezone" "America/Los_Angeles"}}
+
+# Let Janitor Monkey run
+simianarmy.janitor.enabled = {{getv "/simianarmy/janitor/enabled" "true"}}
```

Comment:
I'll revert to false.

But in general, I think it should use the defaults from [SimianArmy](https://github.com/Netflix/SimianArmy) for consistency